README.md

```diff
@@ -51,7 +51,9 @@ module "container_service_cluster" {
 ## Variables
 
 - `vpc_id` - ID of VPC meant to house cluster
-- `ami_id` - Cluster instance Amazon Machine Image (AMI) ID
+- `ami_id` - Cluster instance Amazon Machine Image (AMI) ID. If `lookup_latest_ami` is `true`, this variable will be silently ignored.
+- `ami_owners` - Used with `ami_id`. List of accounts that own AMI with ID `ami_id` (default: [`self`, `amazon`, `aws-marketplace`])
```

Comment:
I'd remove anything in there that ties things with `ami_id`. This should be a standalone variable that applies whether you use `ami_id` or `lookup_latest_ami`.